code/android-core-library/src/main/java/com/adobe/marketing/mobile/ConfigurationExtension.java

```diff
@@ -793,8 +794,7 @@ ConfigurationDownloader getConfigDownloader(final String appId) {
 
         // Build a Configuration Downloader
         try {
-            return new ConfigurationDownloader(getPlatformServices().getNetworkService(),
-                    getPlatformServices().getSystemInfoService(), requestUrlString);
+            return new ConfigurationDownloader(ServiceProvider.getInstance().getNetworkService(), requestUrlString);
```

Comment:
Don't we need to refactor this class to extend Extension instead of InternalModule? Is that happening in another PR?